bin/install_R.sh

```diff
@@ -83,13 +83,15 @@ apt-get --assume-yes install r-recommended
 # not found in Lucid: r-cran-adapt
 
 #Calls R script to do install with feedback to stdout
-#R --no-save < ../app-conf/R/installRpackages.r
+R --no-save < ../app-conf/R/installRpackages.r
```

Comment:
I propose to remove this line here since it has been correctly moved after the apt-get install